### ISSUE TITLE
Rspamd 1.7.0

### ISSUE THREAD

**Issue by hardware** [hardware/mailserver#228]
Rspamd **1.7.0** has been released. The most significant changes are:

* Better machine learning support and embedding of [Lua Torch](http://torch.ch/)
* Language detection support based on ngramms and heuristics
* New `rspamadm configwizard` command for a simple configuration setup
* New statistics model for Redis backend allowing expiration and better analytics
* Improved wizards for statistics conversion and management
* Added automatic corpus test and rescoring utility based on Google Summer of Code 2017 project completed by @cpragadeesh
* New Elasticsearch plugin
* New experimental Reputation plugin

Since Rspamd 1.7, libfann module is deprecated in honor of Lua Torch. This tool allows more powerful neural networks processing.

:warning: 

**Rspamd has switched from the old layout of Redis storage where tokens were stored in two large hash tables: `RSBAYES_SPAM` and `RSBAYES_HAM` to a model where each token is stored separately : `RS_<token_id>` and 2 buckets `S` for spam hits and `H` for ham hits.**

You can convert your old Bayesian statistics in Redis to the new schema using `rspamadm configwizard statistic` command.

```
docker pull hardware/mailserver:1.1-stable && docker-compose up -d
docker exec -ti mailserver rspamadm configwizard statistic
docker-compose stop mailserver; docker-compose rm mailserver; docker-compose up -d
```
_**Info** : you need enough RAM to perform this conversion._

:warning: 

https://rspamd.com/announce/2018/03/12/rspamd-1.7.0.html
https://rspamd.com/doc/migration.html#migration-to-rspamd-170
https://github.com/vstakhov/rspamd/releases/tag/1.7.0

This new version is **now available and ready** in master branch (`1.1-latest` tag) and in **7 days** in stable branch If everything works as expected.

**Comment by hardware** [comment]
I got this error when converting old statistics to the new schema : 

```
docker exec -ti mailserver rspamadm configwizard statistic

You have configured new schema for BAYES_SPAM/BAYES_HAM but your DB has old data
Do you wish to convert data to the new schema?[Y/n]: 
Expire time for new tokens  [default: 100d]: 
converted OK elements from symbol BAYES_SPAM
converted 42386 elements from symbol BAYES_HAM
error converting metadata for symbol BAYES_SPAM
Conversion failed
No changes found, the wizard is finished now
```

Someone else has the same problem ?

**Comment by navossoc** [comment]
Testing it right now...

Just a note for people upgrading from 1.6.x:
> Until you convert your DB the webui is "broken" (bad gateway).

I'll see if its everything ok, then I'll do that ipv6 test again and I let you know about the results.

**Comment by hardware** [comment]
The master branch has been merged into v1.1-stable branch. Rspamd 1.7 is available for everyone.


### Noticeable changes in stable version :

* **Feature** : PostgreSQL support (added by @navossoc)
*  **Update** : Debian 9.4 (Postfix 3.1.8, ClamAV 0.99.4)
* **Update** : Rspamd 1.7 (major release)
  - Don't forget to run `docker exec -ti mailserver rspamadm configwizard statistic` in order to convert your old Bayesian statistics.
*  **Fix** : Traefik backend port is not properly defined https://github.com/containous/traefik/issues/2993#issuecomment-372658641
   - Use `traefik.port` instead of `traefik.backend.port`
*  **Fix** : Vacation replies are not dkim signed https://github.com/hardware/mailserver/issues/227
*  **Fix** : Alias are not sending vacation replies https://github.com/hardware/mailserver/issues/227

**Comment by navossoc** [comment]
Some news: https://github.com/vstakhov/rspamd/releases/tag/1.7.3

As you have mentioned on the issue #230, seems that memory leak has been fixed.
